### DISCUSSION TITLE
Ideas on How to Improve the Developer Experience

### DISCUSSION THREAD

**Discussion by kelanik8** [cortezaproject/corteza-server#1609] [Ideas]

**Comment by kelanik8** [comment]
- Integrate Husky to run code linting before adding a new commit. (The only downside on this is that it would run on all the webapps which would cause a lot of delay when trying to commit changes to a feature)

**Comment by Fajfa** [comment]
Make file with useful commands
`make link`
`make serve compose/admin`

**Comment by tjerman** [comment]
could also have a `make kaboom` (but named differently) that runs all of those link, yarn, ... commands
